doc/api/modules.md

```diff
@@ -159,8 +159,9 @@ require(X) from module at path Y
 3. If X begins with './' or '/' or '../'
    a. LOAD_AS_FILE(Y + X)
    b. LOAD_AS_DIRECTORY(Y + X)
-4. LOAD_NODE_MODULES(X, dirname(Y))
-5. THROW "not found"
+5. LOAD_NODE_MODULES(X, dirname(Y))
+4. LOAD_SELF_REFERENCE(X, dirname(Y))
```

Comment:
Are these entries misnumbered?

Reply:
They are! Let me create a follow-up to fix that.

Reply:
Fix in https://github.com/nodejs/node/pull/29327